frontend/src/components/adminpage/admin-dasboard.js

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import "./admin.css";
+import { Button2 } from "../util/button/button";
+import { Card, ImageHeader, CardBody } from "react-simple-card";
+const AdminDashboard = () => {
+  return (
+    <div>
+      <div className="team container">
```

Comment:
remove the redundant code from here to [here](https://github.com/HITK-TECH-Community/Community-Website/pull/295/files#diff-c7199d15ba5de59466fc8a9c9e7efc6badebbf8ed832f71b4cac4cb15d42947bR88)